config.h

```diff
@@ -0,0 +1,16 @@
+#define MAX_COLOR_PAIR 25
```

Comment:
File name is generic file name and all future configurations might get in this file eventhough they are not related.
It is a better practice to have a meaningful name that reflects the content of the file.

If you observe color related configuration and alert related configuration is already part of this file and both are unrelated!

Reply:
@prithamvshetty , file name is changed as "misalignedConfig.h"